src/components/Auth/SignUpForm.js

```diff
@@ -36,102 +32,106 @@ const SignUpForm = ({ toggleActiveForm }) => {
         setErrorMessage(Object.values(err.response.data).join(''));
       });
   };
+
   if (isLoggedIn) {
     return <Redirect to={referer} />;
   }
+
+  if (auth && auth.authTokens) {
+    return <p>Welcome!</p>;
+  }
+
   return (
-    <>
-      {(auth && auth.authTokens) || isLoggedIn ? (
-        <p>Welcome!</p>
-      ) : (
-        <>
+    <Box
+      component={Form}
+      display="flex"
+      flexWrap="wrap"
+      noValidate
+      autoComplete="off"
+      onSubmit={onSubmit}
+      data-testid="signupForm"
+      validationResolver={validationResolver}
+      defaultValues={defaultValue}
+    >
+      <Box component="h1" fontSize={18}>
+        Create an account
+      </Box>
+
+      <Field
+        as={TextField}
+        fullWidth
+        variant="outlined"
+        margin="dense"
+        name="firstName"
+        label="First Name"
+        id="first-name"
+      />
+      <Field
+        as={TextField}
+        fullWidth
+        variant="outlined"
+        margin="dense"
+        name="lastName"
+        label="Last Name"
+        id="last-name"
+      />
+      <Field
+        as={TextField}
+        fullWidth
+        variant="outlined"
+        margin="dense"
+        name="username"
+        label="Username*"
+        id="username"
+      />
+      <Field
+        as={TextField}
+        fullWidth
+        variant="outlined"
+        margin="dense"
+        name="email"
+        label="Email*"
+        id="email"
+      />
+      <Field
+        as={TextField}
+        fullWidth
+        variant="outlined"
+        margin="dense"
+        name="password"
+        label="Password*"
```

Comment:
why the star here?

Reply:
I think that's to indicate that it's required

Reply:
that sounds like hidden complexity ..

Reply:
I think it's good UI to show the star indicating that the field is required; are there other UI patterns I'm missing?